ttnn/cpp/pybind11/device.cpp

```diff
@@ -143,14 +143,14 @@ void device_module(py::module &m_device) {
         +------------------+------------------------+---------------------+------------------------------+----------+
     )doc",
         py::arg("device_id"),
-        py::arg("num_hw_cqs") = 1,
+        py::arg("num_command_queues") = 1,
         py::arg("l1_small_size") = DEFAULT_L1_SMALL_SIZE,
         py::arg("trace_region_size") = DEFAULT_TRACE_REGION_SIZE,
         py::arg("dispatch_core_type") = tt::tt_metal::DispatchCoreType::WORKER);
     m_device.def(
         "CreateDevices",
-        [](std::vector<int> device_ids, uint8_t num_hw_cqs, size_t l1_small_size, size_t trace_region_size, tt::tt_metal::DispatchCoreType dispatch_core_type) {
-            return tt::tt_metal::detail::CreateDevices(device_ids, num_hw_cqs, l1_small_size, trace_region_size, dispatch_core_type);
+        [](std::vector<int> device_ids, uint8_t num_command_queues, size_t l1_small_size, size_t trace_region_size, tt::tt_metal::DispatchCoreType dispatch_core_type) {
+            return tt::tt_metal::detail::CreateDevices(device_ids, num_command_queues, l1_small_size, trace_region_size, dispatch_core_type);
```

Comment:
does this function `tt::tt_metal::detail::CreateDevices` also needs arg rename?

Reply:
I decided to leave the c++ code the same for this pr.
Was more important to me to get the python apis the same since users can use kwargs in python but not c++, so having it fixed for python was more important.

I think the c++ changes require some additional input from other RT folks if we want a full rename throughout the metal stack or not.